tests/estimators/speech_recognition/test_pytorch_deep_speech.json

```diff
@@ -93,70 +93,70 @@
             4.6668241e-13
         ],
         [
-            -3482.77892371,
-            665.64673575,
-            -116.24408896,
-            265.93803869,
-            1667.02236699,
-            688.33557577,
-            1455.14911883,
-            -3524.90476617,
-            -4082.06471587,
-            -8802.39419605,
-            -277.74274789,
-            2034.54679277,
-            -428.53153241,
-            4114.63683848,
-            1722.53840709,
-            -513.68916798,
-            1159.88786568,
-            7072.47761446,
-            -1963.71829047,
-            382.65287411
-        ],
-        [
-            20992.44844133,
-            3048.78701634,
-            -7849.13725934,
-            15557.59663939,
-            -15760.10725159,
-            -18422.9438386,
-            19132.22699435,
-            6508.51437337,
-            26292.5249963,
-            4232.62414548,
-            -31128.82664215,
-            -2894.85284984,
-            13008.74538039,
-            13845.08921681,
-            17657.67725957,
-            8807.42144017,
-            -16477.89414508,
-            -6977.8092622,
-            -17914.22352666,
-            4086.51150059
-        ],
-        [
-            -1693.10472689,
-            6711.39788693,
-            16480.14166546,
-            -3786.95541286,
-            16448.3969823,
-            -15702.45621671,
-            -26162.89260564,
-            -8979.81601681,
-            9657.87483965,
-            13955.78845296,
-            -7552.01438108,
-            14170.60635269,
-            -24434.37243957,
-            5502.81163675,
-            7171.56926943,
-            -6154.06511686,
-            2483.93980406,
-            -7244.24618697,
-            -10798.70438903,
-            -11129.57632319
+            11.193785,
```

Comment:
Would it be possible to store the expected values as a dictionary to avoid having to index the loaded expected values?

Reply:
Done.